.github/dependabot.yaml

```diff
@@ -0,0 +1,23 @@
+version: 2
+updates:
+  -
+    package-ecosystem: "gomod"
+    directory: "/"
+    schedule:
+      interval: "monthly"
+    labels:
+      - "dependencies"
+  -
+    package-ecosystem: "docker"
+    directory: "/package/"
```

Comment:
@manno can we also add `"/"` for Dockerfile updates, please? I've been enabling this for other repos too.